WordPress/Classes/ViewRelated/System/Notices/NoticeStyle.swift

```diff
@@ -68,6 +68,28 @@ public struct NormalNoticeStyle: NoticeStyle {
     public let dismissGesture: NoticeDismissGesture? = nil
 }
 
+public struct InAppUpdateNoticeStyle: NoticeStyle {
```

Comment:
It looks like it's identical to `QuickStartNoticeStyle`.

Reply:
It is, but I wasn't sure if I'd want to extend it when I implement showing a dismissed notice again after a certain interval. If I don't end up doing that, I'll get rid of it.